.github/CONTRIBUTING.md

```diff
@@ -106,6 +119,24 @@ If the scope of the code change requires it, follow the [Changelog Guide](/.gith
 
 >Note: If you make any changes to the code, run `make format` to automatically format the code according to Go standards.
 
+## Opening a PR
+
+1. Title the PR with a helpful prefix following the pattern `parent/child`, 
+e.g. `builtin/k8s` or `internal/core`
+1. Include helpful information in the description
+   * For a bug fix, explain or show an example of the behavior before and 
+  after the change
+   * If applicable, include information on how to test manually
+1. Assign the `backport/[version].x` label for PRs that should be included 
+in an upcoming .x release (e.g. `backport/0.3.x`)
+1. Assign the `backport/website` label for PRs that should be published 
+immediately to the website
```

Comment:
Nit/question: Assuming the audience of this contributing doc is both public OSS contributors as well as developers at HashiCorp, I feel like maybe we should omit the `backport` labeling parts here, and leave those as a internal task HashiCorp devs are responsible for. What do you think?